rust-src/ed25519_hd_key_derivation/src/lib.rs

```diff
@@ -187,8 +189,8 @@ mod tests {
     fn assert_keys(seed: &str, path: &str, chain_code: &str, private_key: &str, public_key: &str) {
         let seed = hex::decode(seed).unwrap();
         let keys = derive(path, &seed).unwrap();
-        let secret_key = ed25519_dalek::SecretKey::from_bytes(&keys.private_key).unwrap();
-        let public_key_derived = ed25519_dalek::PublicKey::from(&secret_key).to_bytes();
+        let signing = SigningKey::from_bytes(&keys.private_key);
```

Comment:
same comment: I would call it signing_key or sk.

Reply:
Fixed.